h2/utilities.py

```diff
@@ -67,6 +67,13 @@ def secure_headers(headers):
             yield header
 
 
+def extract_method_header(headers):
+
+    for k, v in headers:
+        if k in (b':method', u':method'):
+            return v
```

Comment:
Let's save ourselves time down the road by always ensuring we have a bytes header value.

```python
if not isinstance(v, bytes):
    return v.encode('utf-8')
else:
    return v
```